sections/06_discussion.md

```diff
@@ -49,6 +49,86 @@ with only a couple GPUs.*
 *Some of this is also outlined in the Categorize section.  We can decide where
 it best fits.*
 
+Efficiently scaling deep learning is challenging, and there is a high
+computational cost (e.g., time, memory, energy) associated with training neural
+networks and using them for classification. For these reasons, neural networks
```

Comment:
Fair point. Though the accumulated run time cost may eclipse the design/train time (e.g., a popular and widely used tool), the latter is not distributed, and thus poses a more direct challenge for computational biologists.